File: app/src/main/java/com/shine/semav/numbersystem/common/StringPreferences.java
package com.shine.semav.numbersystem.common;

import android.content.Context;
import android.preference.PreferenceManager;

public class StringPreferences {
    private static final String PREF_STRING_NUMBER = "numberPref";
    private static final String PREF_STRING_FROM = "fromPref";
    private static final String PREF_STRING_TO = "toPref";

    public static String getNumber(Context context){
        return PreferenceManager.getDefaultSharedPreferences(context)
                .getString(PREF_STRING_NUMBER, null);
    }

    public static String getSystemFrom(Context context){
        return PreferenceManager.getDefaultSharedPreferences(context)
                .getString(PREF_STRING_FROM, null);
    }

    public static String getSystemTo(Context context){
        return PreferenceManager.getDefaultSharedPreferences(context)
                .getString(PREF_STRING_TO, null);
    }

    public static void saveStrings(Context context, String number, String systemFrom, String systemTo){
        PreferenceManager.getDefaultSharedPreferences(context)
                .edit()
                .putString(PREF_STRING_NUMBER, number)
                .putString(PREF_STRING_FROM, systemFrom)
                .putString(PREF_STRING_TO, systemTo)
                .apply();
    }


}

File: app/src/main/java/com/shine/semav/numbersystem/common/StringPreferences.kt
package com.shine.semav.numbersystem.common

import android.content.Context
import android.preference.PreferenceManager

/**
 * Created by semav on 18.09.2016.
 */
object StringPreferences {
    private val PREF_STRING_NUMBER = "numberPref"
    private val PREF_STRING_FROM = "fromPref"
    private val PREF_STRING_TO = "toPref"

    fun getNumber(context: Context): String? = PreferenceManager.getDefaultSharedPreferences(context).getString(PREF_STRING_NUMBER, "")


    fun getSystemFrom(context: Context): String? = PreferenceManager.getDefaultSharedPreferences(context).getString(PREF_STRING_FROM, "")


    fun getSystemTo(context: Context): String? = PreferenceManager.getDefaultSharedPreferences(context)
            .getString(PREF_STRING_TO, "")


    fun saveStrings(context: Context, number: String, systemFrom: String, systemTo: String) {
        PreferenceManager.getDefaultSharedPreferences(context)
                .edit()
                .putString(PREF_STRING_NUMBER, number)
                .putString(PREF_STRING_FROM, systemFrom)
                .putString(PREF_STRING_TO, systemTo)
                .apply()
    }


}
